app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -47,6 +47,9 @@ interface ProjectViewModel {
         /** Call when the native back project button is clicked.  */
         fun nativeCheckoutBackProjectButtonClicked()
 
+        /** Call when the native manage pledge button is clicked  */
+        fun nativeCheckoutManagePledgeButtonClicked()
```

Comment:
I don't think this is used in the `ProjectActivity`. We should probably rename `nativeCheckoutBackProjectButtonClicked` to be something like `nativeProjectActionButtonClicked` to be more accurate.